sass/compat/plugins/woo-carrier-agents/_checkout-shipping-methods.scss

```diff
@@ -0,0 +1,16 @@
+// ------------------------------------
+// CHECKOUT SHIPPING METHODS
+// ------------------------------------
+
+body.woocommerce-checkout .fc-content .carrier-agent {
+	padding-#{ $alignment-left }: 0;
+}
+
+body.woocommerce-checkout .fc-content .carrier-agent .carrier-agent-input input[type="radio"] {
+	position: static !important; // Needed to override FC styles
+	margin-top: remsize( 0.5rem ) !important; // Needed to override FC styles
+	margin-#{ $alignment-right }: remsize( 1rem ) !important; // Needed to override FC styles
+	width: auto;
+	height: auto;
```

Comment:
- Does this work for various themes?

Reply:
It's to override the FC's `width: 1px; height: 1px;`. I checked a couple of themes where this works, but it's possible that it would be overridden in some other themes. Are there any better ways to do this?

Reply:
@vlad-dedikov I see. The original styles from Fluid Checkout are targeting the radio button related to the shipping method itself. What we could do in this case would be to change the original styles to target specifically the radio button right under the shipping method option with `.shipping-method__option > input[type="radio"] {` instead of the current `.shipping-method__option input[type="radio"] {`.

I made this change on the original plugin files, see if it solves the problem for this plugin. You can probably remove all these styles targeting the radio buttons since the other properties will likely not be set to these elements anymore.